cli/azd/internal/appdetect/spring_boot.go

```diff
@@ -249,8 +297,26 @@ func detectEventHubsAccordingToSpringCloudStreamKafkaMavenDependency(
 	}
 }
 
+func detectEventHubsAccordingToSpringKafkaMavenDependency(azdProject *Project, springBootProject *SpringBootProject) {
+	var targetGroupId = "org.springframework.kafka"
+	var targetArtifactId = "spring-kafka"
+	if hasDependency(springBootProject, targetGroupId, targetArtifactId) {
+		newDep := AzureDepEventHubs{
+			// eventhubs name is empty here because no configured property
```

Comment:
Could you please write a doc about the event hub name detecting? Of course, it can be done after current PR finished. In my imagination, it should be like this:

Table: Event Hub names detected from project
| Maven dependency name | Application Property name | Java Code |
| -----------------------------|-------------------------------| ------------|
| dependency-xxx                | property-xxx (implemented)                        | code-xxx(Not implemented) |
| dependency-yyy                | Not exist                        | not exist |


And you can choose emoji to make above table look better:
https://github.com/ikatyang/emoji-cheat-sheet/blob/master/README.md

Now you can put this doc here: https://github.com/azure-javaee/azure-dev/wiki

Reply:
Sure, will track with this item https://devdiv.visualstudio.com/DevDiv/_workitems/edit/2330465.

Reply:
Thank you.